packages/taco/src/conditions/base/contract.ts

```diff
@@ -70,6 +70,31 @@ const functionAbiSchema = z
     },
   );
 
+function toJsonAbiFormat(humanReadableAbi: string): any {
+  const abiWithoutFunctionKeyword = humanReadableAbi.replace(
+    /^function\s+/,
+    '',
+  );
+  const fragment = ethers.utils.FunctionFragment.from(
+    abiWithoutFunctionKeyword,
+  );
+  const jsonAbi = JSON.parse(fragment.format(ethers.utils.FormatTypes.json));
+
+  delete jsonAbi.constant;
+  delete jsonAbi.payable;
```

Comment:
Why do we delete these fields? Can we use destructuring and only pick relevant fields instead?